### ISSUE TITLE
Incompatibility with jurassic craft

### ISSUE THREAD

**Issue by Wachecheiro** [DrZhark/mocreatures#226]
Hello, it seems that Custom mob spawner somehow overlaps Jurassic craft, in config in creatures it appears "ORG.cfg" but it seems that it has no effect to change the values there, by default many spawn values appear in "false", setting everything to "true" does not have any effect even having previously enabled the generation of dinosaurs within the Jurassic craft menu "ingame", only 1 or 2 dinosaurs seem to work "without touching any of the settings", if I'm not mistaken dilophosaurus is one of them (if not the only one), also most of them bring "S: type = UNDEFINED" .., the only way I have dinosaurs reappear is by disabling them within the game and re-enabling them (and this doesn't always work), but it seems at the same time to disable Mo 'Creatures because it fills absolutely everything with dinosaurs.
In short Custom mob spawner seems to detect Jurasicc craft but it doesn't work together

**Comment by MagmaSlime123** [comment]
Custom Mob Spawner mod is VERY broken, as a whole. I have an issue related to this as well.

Doubt this mod will ever be developed again. Pointless even being on CurseForge anyways.
